### PR TITLE
discourage use of ibm_container_bind_service

### PR DESCRIPTION
**Pull request by bevicted**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

We discourage using `ibm_container_bind_service` as it will be deprecated.
Replicating what `ibm_container_bind_service` does in several smaller steps.

### CODE REVIEW

File: website/docs/r/container_bind_service.html.markdown [hunk 1]
```diff
@@ -8,6 +8,11 @@ description: |-
 ---
 
 # ibm_container_bind_service
+
+> [!CAUTION]
+> This resource is not reliable and will be deprecated.
+It is highly recommended to bind your services according to [this guide](I'm unsure how to link to the newly written guide)
```
Comment (bevicted) [inline website/docs/r/container_bind_service.html.markdown:14]: I will need some guidance here on how to link another file

**Comment by bevicted** [comment]
I thought it better fit these new instructions to be included in a "guide" section than just including it in the file for `ibm_container_bind_service` however I did not find the existing `guides/custom-service-endpoints.html.md` in the online docs, so please advise.

File: website/docs/r/container_bind_service.html.markdown (hunk 1 above)
Comment (hkantare) [reply website/docs/r/container_bind_service.html.markdown:14]: which link you mean here .Link outside this files?

Comment (bevicted) [reply website/docs/r/container_bind_service.html.markdown:14]: The new guide I've added, in `website/docs/guides/binding-services.html.md`

Comment (bevicted) [reply website/docs/r/container_bind_service.html.markdown]: Done in [b10b72a](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5588/commits/b10b72a27457bcc8b838cf187399893789a6b500)

**Review by hkantare — APPROVED** [review]

**Pull request merged**